easy_study/lib/presenter/TimeTracking.dart

```diff
@@ -0,0 +1,122 @@
+import 'dart:async';
+
+import 'package:easy_study/Database/DBHelper.dart';
+import 'package:easy_study/model/Subject.dart';
+import 'package:flutter/material.dart';
+
+class TimeTracking extends StatefulWidget {
```

Comment:
I found a bug or a wanted feature:
When I stop like a minute and then hit the pause button and then retry stopping time -> the old time isn't added to the overall time.

We should add a pause button, and when the start button got pressed the pause button should be shown and the start button should be hidden.
The square button shouldn't be a square button, maybe a submit tracked time button or something like this.
Maybe also show a snack bar that the user spent like 15 minutes to learn and a happy face button or a quote that he earned a pause.
But I like it a lot!

Reply:
https://github.com/mobileappdevhm19/apps4students-team-4/issues/21